cineast-core/src/main/java/org/vitrivr/cineast/core/util/MultiTracker.java

```diff
@@ -0,0 +1,82 @@
+package org.vitrivr.cineast.core.util;
+
+import boofcv.abst.tracker.TrackerObjectQuad;
+import boofcv.factory.tracker.FactoryTrackerObjectQuad;
+import boofcv.struct.image.GrayU8;
+import georegression.struct.shapes.Quadrilateral_F64;
+import org.vitrivr.cineast.core.data.Pair;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class MultiTracker {
```

Comment:
Same comment as in the `HungarianAlgorithm`, documentation putting this into context would be appreciated

Reply:
Is resolved in the new commit. I additionally made clarifying comments in other classes I created.